<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRSRunDialog</class>
 <widget class="QDialog" name="CRSRunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robot-Airship Capture - Run Console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QRadioButton" name="static_sim_check">
        <property name="text">
         <string>Static Simulation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dynamic_sim_check">
        <property name="text">
         <string>Dynamic Simulation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="live_sim_check">
        <property name="text">
         <string>Live Dry-run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="live_run_check">
        <property name="text">
         <string>Live Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="start_button">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: navy; background-color: lime;</string>
           </property>
           <property name="text">
            <string>START
PLANNING</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: red; background-color: orange;</string>
           </property>
           <property name="text">
            <string>STOP
PLANNING</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="proceed_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: navy; background-color: lime;</string>
           </property>
           <property name="text">
            <string>LAUNCH
SOLUTION</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="abort_button">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: yellow; background-color: red;</string>
           </property>
           <property name="text">
            <string>ABORT!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="status_text">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="init_label">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: red;</string>
              </property>
              <property name="text">
               <string>Initializing...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="planning_label">
              <property name="styleSheet">
               <string notr="true">color: red;</string>
              </property>
              <property name="text">
               <string>Planning...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="launch_label">
              <property name="styleSheet">
               <string notr="true">color: red;</string>
              </property>
              <property name="text">
               <string>Launching...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="capture_label">
              <property name="styleSheet">
               <string notr="true">color: red;</string>
              </property>
              <property name="text">
               <string>Captured!</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
